### DISCUSSION TITLE
Move official proposal discussions to their own category?

### DISCUSSION THREAD

**Discussion by tristanbiesecker** [modular/modular#339] [General]
What do folks think about creating a new category for official proposal discussions to live?
<img width="350" alt="Screenshot 2023-06-06 at 3 36 19 PM" src="https://github.com/modularml/mojo/assets/12146879/9953a710-35d1-48db-bcb6-fbbd30f48389">

**Comment by lattner** [comment]
@goldiegadde WDYT? I don't know how any of this works :-) :-)

**Comment by strangemonad** [comment]
@tristanbiesecker are you suggesting a new "Proposals" label be strictly for discussion of existing proposals? I've been using "Ideas" to float half baked things that could become official proposals.

Overall in favor of having an explicit label because it makes searching and filtering easier.

**Comment by tristanbiesecker** [comment]
@strangemonad sorry for the late reply here, but yes! More specifically, it would be reserved for the more "formal" proposals like ones already in the [proposals](https://github.com/modularml/mojo/tree/main/proposals) directory. And I agree with your approach of using "Ideas" for thoughts that may not be fully fleshed out yet.

**Comment by goldiegadde** [comment]
Sorry for the late response on this one, working on how best to group this in Discussions, will update soon. Thanks for your feedback and input.
